models/articles.js

```diff
@@ -0,0 +1,20 @@
+const db = require('../db/connection')
+
+exports.fetchArticles = () => {
+  return db.query(`
+    SELECT articles.*, COUNT(comments.comment_id) AS comment_count
```

Comment:
we shouldn't be selecting the review_body from articles, how can we leave that out?